src/rogue/interfaces/memory/TcpServer.cpp

```diff
@@ -103,10 +103,14 @@ rim::TcpServer::TcpServer (std::string addr, uint16_t port) {
 
 //! Destructor
 rim::TcpServer::~TcpServer() {
-  this->close();
+  this->stop();
 }
 
 void rim::TcpServer::close() {
+    this->stop();
```

Comment:
Need to add a print statement about how  `close()` is deprecated and will be removed in the future and should use `stop()` instead